### PR TITLE
adding Healey 1970

### PR DESCRIPTION
**Pull request by SimonGreenhill**
# Pull request checklist

- [X] add new concept list
- [ ] add new metadata
- [ ] add new Concepticon concept sets
  * [ ] checked whether the new concept(s) can be applied to existing lists with
    `concepticon notlinked --gloss "NEW_GLOSS"`
- [ ] add new Concepticon concept relations
- [ ] refine existing Concepticon concept set mappings
- [ ] refine Concepticon glosses
- [ ] refine Concepticon concept relations
- [ ] refine Concepticon concept definitions
- [ ] retire data

## Additional information

New concept list, scanned list attached here:

[Healey1970.pdf](https://github.com/concepticon/concepticon-data/files/11073869/Healey1970.pdf)

### CODE REVIEW

File: concepticondata/conceptlists/Healey-1970-364.tsv
Comment (MuffinLinwist) [inline concepticondata/conceptlists/Healey-1970-364.tsv:57]: I'd map this as 355 CUT DOWN, as previous mappings.

File: concepticondata/conceptlists/Healey-1970-364.tsv
Comment (MuffinLinwist) [inline concepticondata/conceptlists/Healey-1970-364.tsv:73]: I'd also unmapped because the concept on the list is more specific and because of previous mappings.

File: concepticondata/conceptlists/Healey-1970-364.tsv
Comment (MuffinLinwist) [inline concepticondata/conceptlists/Healey-1970-364.tsv:18]: I'd unmapped because a flying fox is a type of fruit bat? I'm not sure about this.

File: concepticondata/conceptlists/Healey-1970-364.tsv
Comment (MuffinLinwist) [inline concepticondata/conceptlists/Healey-1970-364.tsv:128]: I'd map this as 1752 LEAVE, according to previous mappings.

**Comment by AnnikaTjuka** [comment]
@SimonGreenhill I'll moderate the PR after the release. I hope it's okay if we don't include the list in 3.1.

File: concepticondata/conceptlists/Healey-1970-364.tsv
Comment (MuffinLinwist) [inline concepticondata/conceptlists/Healey-1970-364.tsv:165]: Perhaps we can map this as 945 SPEAR?

File: concepticondata/conceptlists/Healey-1970-364.tsv
Comment (MuffinLinwist) [inline concepticondata/conceptlists/Healey-1970-364.tsv:278]: 1503 SALIVA

File: concepticondata/conceptlists/Healey-1970-364.tsv
Comment (MuffinLinwist) [inline concepticondata/conceptlists/Healey-1970-364.tsv:329]: Perhaps 3236 TWIST (TOGETHER)?

File: concepticondata/conceptlists/Healey-1970-364.tsv
Comment (MuffinLinwist) [inline concepticondata/conceptlists/Healey-1970-364.tsv:346]: 1839 CRY

File: concepticondata/conceptlists/Healey-1970-364.tsv
Comment (MuffinLinwist) [inline concepticondata/conceptlists/Healey-1970-364.tsv:363]: 1207 YOUNG

**Review by MuffinLinwist — COMMENTED** [review]
@AnnikaTjuka I was just finishing writing my review. So maybe it can stay there until your revision after the release.

File: concepticondata/conceptlists/Healey-1970-364.tsv
Comment (SimonGreenhill) [reply concepticondata/conceptlists/Healey-1970-364.tsv:18]: this is likely to be a flying fox (https://en.wikipedia.org/wiki/Great_flying_fox) rather than a fruit bat (https://en.wikipedia.org/wiki/Bulmer%27s_fruit_bat), as the author seems to use these synonymously

File: concepticondata/conceptlists/Healey-1970-364.tsv
Comment (SimonGreenhill) [reply concepticondata/conceptlists/Healey-1970-364.tsv:129]: ok, but I think of "extinguish" as a more active/transitive thing t

File: concepticondata/conceptlists/Healey-1970-364.tsv
Comment (SimonGreenhill) [reply concepticondata/conceptlists/Healey-1970-364.tsv:165]: I can't tell if this is a verb or a noun, which is why I left it alone

**Comment by SimonGreenhill** [comment]
Thanks @MuffinLinwist  -- most changes made, except for the ones I commented on.

**Comment by AnnikaTjuka** [comment]
Thanks, @MuffinLinwist and @SimonGreenhill! Would you have time to do the second review for the list, @Tarotis?

File: concepticondata/conceptlists/Healey-1970-364.tsv
Comment (Tarotis) [inline concepticondata/conceptlists/Healey-1970-364.tsv:18]: Would it be possible to change the gloss, though? If we keep it like this, it reads as if "bat" and "fruit" were two separate meanings

File: concepticondata/conceptlists/Healey-1970-364.tsv
Comment (AnnikaTjuka) [reply concepticondata/conceptlists/Healey-1970-364.tsv:169]: Good catch! I checked and we have a few other lists where we mapped both glosses (_go away_ and _leave_) to LEAVE. @Tarotis Could you add an issue where we discuss whether we want to split the concept? @SimonGreenhill I'd leave this instance as is for now.

**Comment by AnnikaTjuka** [comment]
@Tarotis I just went through all your comments. I'd generally refrain from changing the glosses if they are in the original list. If we notice a lot of mistakes in a list, we usually add an extra column where we correct the glosses. But it must be transparent that we are making the changes and that they do not correspond to the original list.

File: concepticondata/conceptlists/Healey-1970-364.tsv
Comment (Tarotis) [reply concepticondata/conceptlists/Healey-1970-364.tsv:18]: Based on Annikas comment ,the gloss should not be changed :)

File: concepticondata/conceptlists/Healey-1970-364.tsv
Comment (SimonGreenhill) [reply concepticondata/conceptlists/Healey-1970-364.tsv:5]: no, the words listed in the languages here seem to contain both "arm" and "arm, hand" glosses.

File: concepticondata/conceptlists/Healey-1970-364.tsv
Comment (SimonGreenhill) [reply concepticondata/conceptlists/Healey-1970-364.tsv:40]: I don't know -- the lexemes in the underlying languages are different. I guess it's the distinction between whether the rope gets broken, or whether someone breaks the rope. The concepticon gloss is "The breaking of a thread or strap" which I think matches "(rope) breaks" better?

Comment (SimonGreenhill) [reply concepticondata/conceptlists/Healey-1970-364.tsv:40]: (so I've removed the mapping on L36)

File: concepticondata/conceptlists/Healey-1970-364.tsv
Comment (SimonGreenhill) [reply concepticondata/conceptlists/Healey-1970-364.tsv:101]: done.

File: concepticondata/conceptlists/Healey-1970-364.tsv
Comment (SimonGreenhill) [reply concepticondata/conceptlists/Healey-1970-364.tsv:225]: yes, it seems to be part of other verbs too like 'to gather'

File: concepticondata/conceptlists/Healey-1970-364.tsv
Comment (SimonGreenhill) [reply concepticondata/conceptlists/Healey-1970-364.tsv:270]: yes, some of the lexemes are glossed as "sole, only"

**Comment by SimonGreenhill** [comment]
worked through @Tarotis suggestions.

**Comment by AnnikaTjuka** [comment]
Thanks, @SimonGreenhill! I'll do a final check later today.

File: concepticondata/conceptlists/Healey-1970-364.tsv
Comment (AnnikaTjuka) [inline concepticondata/conceptlists/Healey-1970-364.tsv:145]: Could we map this to the narrow concept 2473	HIS (GENITIVE OF HE)?

File: concepticondata/conceptlists/Healey-1970-364.tsv
Comment (AnnikaTjuka) [inline concepticondata/conceptlists/Healey-1970-364.tsv:213]: Due to the mapping below, I'd map to 2112	OLD (AGED).

File: concepticondata/conceptlists/Healey-1970-364.tsv
Comment (AnnikaTjuka) [inline concepticondata/conceptlists/Healey-1970-364.tsv:228]: Could this be mapped to 658	RAIN (PRECIPITATION)?

**Review by AnnikaTjuka — COMMENTED** [review]
Just three more suggestions but then everything should be ready to be merged. @Tarotis We'll need your approval of the requested changes before merging.

**Review by Tarotis — APPROVED** [review]

File: concepticondata/conceptlists/Healey-1970-364.tsv
Comment (SimonGreenhill) [reply concepticondata/conceptlists/Healey-1970-364.tsv:145]: ok

File: concepticondata/conceptlists/Healey-1970-364.tsv
Comment (SimonGreenhill) [reply concepticondata/conceptlists/Healey-1970-364.tsv]: ok

**Comment by SimonGreenhill** [comment]
all Annika's changes have been made.

**Review by AnnikaTjuka — APPROVED** [review]
Many thanks, @SimonGreenhill! Feel free to merge.

**Comment by SimonGreenhill** [comment]
thanks Annika, merged.

**Pull request merged**
